### DISCUSSION TITLE
Save a specific date

### DISCUSSION THREAD

**Discussion by Kwakoxium** [Tyrrrz/DiscordChatExporter#661] [General]
Hi!

I wanted to know if there were any plans to add a filter to save posts from a specific date rather than before or after it. This could be practical in some cases. 

Thanks for your help!

**Comment by Tyrrrz** [comment]
What do you mean by "specific date"? Specific date & time or just date? If the latter, you can already do that -- just combine before & after parameters to narrow it down to one day.

**Reply by Kwakoxium** [reply]
Only the date, a specific day, but regardless of the time. I thought it was not possible.

Thank you for the answer !

**Reply by Kwakoxium** [reply]
Hi !

I finally tried by choosing a random date, July 22. So I selected in the filters after July 21 and before July 23, so there should have been only July 22 in the saved messages. However, those of July 21 were also present (not those of the 23).

I don't know if I did something wrong.

Thank you !

**Reply by Tyrrrz** [reply]
Use After July 22 and Before July 23

**Reply by Kwakoxium** [reply]
In this way, it works correctly.

**Comment by NealWalters** [comment]
Where do we put these filters?  I looked at setting, and didn't see anything like this? 
Also would be cool to remember the last datetime downloaded, and only retrieve since then.
